Classes/MapEditor/MapEditor.cs

```diff
@@ -455,21 +453,22 @@ public void Dispose()
             return;
         }
 
-        List<Note> mirroredSelection = currentMapDifficulty.selectedNotes
-            .Select(note => new Note(note.beat, 3 - note.col))
-            .ToList();
+        var mirroredSelection = currentMapDifficulty.selectedNotes
+            .Select(note => new Note(note.beat, 3 - note.col));
 
         UpdateNotes(mirroredSelection, currentMapDifficulty.selectedNotes);
     }
     public void ShiftSelectionByBeat(MoveNote direction) {
         if (currentMapDifficulty == null) {
             return;
         }
-        List<Note> movedNotes = currentMapDifficulty.selectedNotes
-            .Select(n => {
+        var movedNotes = currentMapDifficulty.selectedNotes
+            .Select(n =>
+            {
```

Comment:
if that is the convention, maybe we should apply it everywhere to keep it consistent. we can run an automatic formatting to convert all files once.